cinema/views.py

```diff
@@ -48,6 +80,35 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
     queryset = MovieSession.objects.all()
     serializer_class = MovieSessionSerializer
 
+    @staticmethod
+    def _params_to_ints(qs) -> list:
+        """Converts a list of string IDs to a list of integers"""
+        return [int(str_id) for str_id in qs.split(",")]
+
+    def get_queryset(self):
+        queryset = self.queryset
+        movie = self.request.query_params.get("movie") or None
+        date = self.request.query_params.get("date") or None
```

Comment:
you dont need None there:
dictionary.get(keyname, value)
value | Optional. A value to return if the specified key does not exist.Default value None

Reply:
I know, but I kinda like it better when it explicitly says "None", not sure whether it is a problem or not 🙃